src/com/severalcircles/colors/commands/CommandNedry.java

```diff
@@ -28,7 +28,7 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
         target.getWorld().spawnEntity(player.getLocation(), EntityType.PRIMED_TNT);
         target.getWorld().spawnEntity(player.getLocation(), EntityType.PRIMED_TNT);
         int chatspam = 0;
-        while (chatspam < 3000) {
+        while (chatspam < 69) {
```

Comment:
You might be able to simplify this with a for loop instead of a while loop.

Reply:
Made it a loop